README.md

```diff
@@ -0,0 +1,26 @@
+This is a starter template for [Ionic](http://ionicframework.com/docs/) projects.
+
+## How to use this template
+
+*This template does not work on its own*. The shared files for each starter are found in the [ionic2-app-base repo](https://github.com/ionic-team/ionic2-app-base).

```

Comment:
Not sure why you say this template does not work on its own. I did the following which is okay.

```
project_dir# ionic start myTabs tabs
project_dir# ionic cordova platform add ios
```

Then I run the project using Xcode, because `ionic cordova run ios` does not work for me:

```
[....] Waiting up to 1 seconds for iOS device to be connected
[....] Found abcd (D10AP, iPhone 7, iphoneos, arm64) a.k.a. 'phonename' connected through USB.
unzip:  cannot find or open /Users/cheungpat/projects/skygear-demo/skygear-ionic/platforms/ios/build/device/MyApp.ipa, /Users/cheungpat/projects/skygear-demo/skygear-ionic/platforms/ios/build/device/MyApp.ipa.zip or /Users/cheungpat/projects/skygear-demo/skygear-ionic/platforms/ios/build/device/MyApp.ipa.ZIP.
Error: Cannot read property 'replace' of undefined
```